src/desktop/components/split_test/middleware.coffee

```diff
@@ -1,18 +1,21 @@
 SplitTest = require './server_split_test.coffee'
 runningTests = require './running_tests'
 qs = require 'qs'
+sd = require('sharify').data
 
 module.exports = (req, res, next) ->
   for key, configuration of runningTests
     unless res.locals.sd[key?.toUpperCase()]?
       test = new SplitTest req, res, configuration
       res.locals.sd[key.toUpperCase()] = test.outcome()
+      sd[key.toUpperCase()] = test.outcome()
```

Comment:
Necessary and based on https://github.com/artsy/force/pull/7401 . Server-side (if someone navigates to /works-for-sale directly), this makes the value available in sharify (as well as the locals).